save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/test/analysis/results/RegularTest.kt

```diff
@@ -0,0 +1,11 @@
+package com.saveourtool.save.test.analysis.results
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@SerialName("RegularTest")
```

Comment:
why we need to specify serial name explicitly, does it work out of box by class name?

Reply:
Removed: b2090794a9d2a19372f64f40437251d8a5fe743c